webapp/graphql/Registration.js

```diff
@@ -7,6 +7,7 @@ export const SignupVerificationMutation = gql`
     $password: String!
     $about: String
     $termsAndConditionsAgreedVersion: String!
+    $locale: String
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-17T10:50:32Z" title="Thursday, October 17th 2019, 12:50:32 pm +02:00">Oct 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,6 +7,7 @@ export const SignupVerificationMutation = gql`
     $password: String!
     $about: String
     $termsAndConditionsAgreedVersion: String!
+    $language: String
```


This is going in the right direction. Could you also add `locale` as allowed input parameter to `UpdateUser`, please?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-17T16:34:14Z" title="Thursday, October 17th 2019, 6:34:14 pm +02:00">Oct 17, 2019</time>_

---


mit dem nächsten PR würde ich das gern + anwendungsfall erledingen um die sache nicht wieder so in die länge zu ziehen

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-17T22:34:56Z" title="Friday, October 18th 2019, 12:34:56 am +02:00">Oct 18, 2019</time>_

---


@ogerly OK, could you rename `language` to `locale`? I'm OK to merge this then.

@mattwr18 what is the better name here? :point_up: I researched `locale` a little and it seems it denominates the language code + country. https://en.wikipedia.org/wiki/Language_code
So it's actually `en-US` and not only `en`. https://en.wikipedia.org/wiki/Locale_(computer_software) We only use the language code. `locale` still good?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-18T10:18:15Z" title="Friday, October 18th 2019, 12:18:15 pm +02:00">Oct 18, 2019</time>_

---


@roschaefer **language** to **_locale_** change